spec/requests/api/v2/providers/sites_spec.rb

```diff
@@ -321,6 +325,12 @@ def perform_site_update
           .to(region_code)
       end
 
+      it "updates the urn of the site" do
+        expect { subject }.to change { site1.reload.urn }
+                                .from(site1.urn)
```

Comment:
Could we sort out the indentation for this?

Reply:
sure